app/models/claims/allocation_filters.rb

```diff
@@ -51,7 +51,7 @@ def all_interim_fees
       interim_fee_types = ['Effective PCMH', 'Trial Start', 'Retrial New Solicitor', 'Retrial Start']
       where(type: 'Claim::InterimClaim')
         .joins(:fees)
-        .where('"fees"."fee_type_id" IN (?)', Fee::InterimFeeType.where(description: interim_fee_types).pluck(:id))
+        .where('"fees"."fee_type_id" IN (?)', Fee::InterimFeeType.where(description: interim_fee_types).select(:id))
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3621/files#r532750493)